docs/docs/configuration/alpha_config.md

```diff
@@ -307,15 +331,13 @@ Provider holds all configuration for a single provider
 | `name` | _string_ | Name is the providers display name<br/>if set, it will be shown to the users in the login page. |
 | `caFiles` | _[]string_ | CAFiles is a list of paths to CA certificates that should be used when connecting to the provider.<br/>If not specified, the default Go trust sources are used instead |
 | `loginURL` | _string_ | LoginURL is the authentication endpoint |
+| `loginURLParameters` | _[map[string]github.com/oauth2-proxy/oauth2-proxy/v7/pkg/apis/options.LoginURLParameter](#loginurlparameter)_ | LoginURLParameters defines the parameters that can be passed from the start URL to the IdP login URL |
```

Comment:
Hmm, why has it come up like that, normally it knows about types in the same package, will need to look into that